packages/rx_bloc_cli/mason_templates/bricks/feature_google_login/__brick__/lib/lib_auth_google/repositories/google_auth_repository.dart

```diff
@@ -0,0 +1,22 @@
+import '../../base/common_mappers/error_mappers/error_mapper.dart';
+import '../../lib_auth/models/auth_token_model.dart';
+import '../data_sources/remote/google_auth_data_source.dart';
+import '../models/google_auth_request_model.dart';
+
+class GoogleAuthRepository {
+  GoogleAuthRepository(this._googleAuthDataSource, this._errorMapper);
+  final GoogleAuthDataSource _googleAuthDataSource;
+  final ErrorMapper _errorMapper;
+
+  Future<AuthTokenModel> googleAuth(
+          {String? email, String? token, String? refreshToken}) =>
+      _errorMapper.execute(
+        () => _googleAuthDataSource.googleAuth(
+          GoogleAuthRequestModel(
+            email: email,
+            token: token,
+            refreshToken: refreshToken,
+          ),
+        ),
+      );
```

Comment:
I believe that we don't need and don't even have refresh token on this step of authentication process.